net/frr/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/dialogEditBGPRouteMaps.xml

```diff
@@ -29,12 +29,20 @@
     <type>select_multiple</type>
     <style>tokenize</style>
     <allownew>true</allownew>
-    <help>Select the AS-Path list</help>
+    <help>Select the AS-Path List.</help>
+  </field>
+  <field>
+    <id>routemap.match2</id>
```

Comment:
I would use as_path_list_match to make it more clear

Reply:
I did a no-scaler mistake at the time implementing it that I only thought of as-path lists, so I just called it match. But matching can be used for anything .. so the question is, count new features up match2, match3, match4 .. of keep match for as-path and name new stuff by there meaning, like prefix_list_match. I'm open for both, but first having "math" and then having specific meanings would make it also fuzzy to read. Opinions?

Reply:
It is interesting to think that the match has several possibilities, I just indicated the prefix-list because it is one of the most used. But if the problem is how to structure the code, it might be interesting to think about it.

Here are examples:

https://www.nongnu.org/quagga/docs/docs-multi/Route-Map-Match-Command.html#Route-Map-Match-Command

Reply:
Mostly the match community, which would be my next feature request.